unlock-app/src/components/interface/checkout/alpha/Checkout/Payment.tsx

```diff
@@ -196,6 +196,35 @@ export function Payment({ injectedProvider, checkoutService, onClose }: Props) {
               />
             </div>
           </button>
+          {isReceiverAccountOnly && (
+            <button
+              disabled={!fiatPricing?.creditCardEnabled || isLoading}
+              onClick={(event) => {
+                event.preventDefault()
+                send({
+                  type: 'SELECT_PAYMENT_METHOD',
+                  payment: {
+                    method: 'claim',
+                  },
+                })
+                send('CONTINUE')
+              }}
+              className="border flex flex-col w-full border-gray-400 space-y-2 cursor-pointer shadow p-4 rounded-lg group hover:bg-gray-50 disabled:cursor-not-allowed disabled:opacity-60 disabled:hover:bg-white"
+            >
+              <h3 className="font-bold"> Claim membership for free </h3>
+              <div className="flex items-center w-full justify-between">
+                <div className="text-sm text-gray-500">
+                  We will pay for it out of our pocket. Enjoy.
```

Comment:
I worry that "pay for it" migth be confusing since this is free. What about this?

```suggestion
                  We will airdrop this free membership to you!
```